proto/dymensionxyz/dymension/dymns/params.proto

```diff
@@ -0,0 +1,190 @@
+  ];
+
+  // Price extends yearly, after first year
+  string price_extends = 6 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"price_extends\"",
+    (gogoproto.nullable) = false
+  ];
+
+  // Denom of the price setup
+  string price_denom = 7 [
+    (gogoproto.moretags) = "yaml:\"price_denom\""
+  ];
+
+  // Minimum price of an offer.
+  // Mostly used to prevent spamming and abusing store with low price offers,
+  // so the value should not be so low.
+  string min_offer_price = 8 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"min_offer_price\"",
+    (gogoproto.nullable) = false
+  ];
+}
+
+message ChainsParams {
+  // Chain-IDs and their aliases, used for UX improvement, used in front-end.
+  repeated AliasesOfChainId aliases_of_chain_ids = 1 [
+    (gogoproto.moretags) = "yaml:\"aliases_of_chain_ids\"",
+    (gogoproto.nullable) = false
+  ];
+
+  // Chain-IDs that are using coin-type 60. Used to ensure address conversion.
+  repeated string coin_type_60_chain_ids = 2 [
+    (gogoproto.customname) = "CoinType60ChainIds",
+    (gogoproto.moretags) = "yaml:\"coin_type_60_chain_ids\""
+  ];
+}
+
+// AliasesOfChainId defines the multiple-aliases of a chain id.
+message AliasesOfChainId {
+  // Chain-ID which owned the aliases.
+  string chain_id = 1 [
+    (gogoproto.moretags) = "yaml:\"chain_id\""
+  ];
+
+  // Multiple aliases of the chain id for UX improvement, used in front-end.
+  repeated string aliases = 2 [
+    (gogoproto.moretags) = "yaml:\"aliases\""
+  ];
+}
+
+message MiscParams {
+  // Identifier of the begin epoch hook.
+  string begin_epoch_hook_identifier = 1 [
+    (gogoproto.moretags) = "yaml:\"begin_epoch_hook_identifier\""
+  ];
+
+  // Identifier of the end epoch hook.
+  string end_epoch_hook_identifier = 2 [
+    (gogoproto.moretags) = "yaml:\"end_epoch_hook_identifier\""
+  ];
+
+  // Amount of time that the former owner of an expired Dym-Name can renew it before completely lost.
+  google.protobuf.Duration grace_period_duration = 3 [
+    (gogoproto.nullable) = false,
+    (gogoproto.stdduration) = true,
+    (gogoproto.moretags) = "yaml:\"grace_period_duration\""
+  ];
+
+  // Amount of time of a Sell-Order from created to expired.
+  google.protobuf.Duration sell_order_duration = 4 [
+    (gogoproto.nullable) = false,
+    (gogoproto.stdduration) = true,
+    (gogoproto.moretags) = "yaml:\"sell_order_duration\""
+  ];
+
+  // Amount of time to keep history version of closed SO before it is deleted,
+  // based on expiry date and block time.
+  google.protobuf.Duration preserved_closed_sell_order_duration = 5 [
+    (gogoproto.nullable) = false,
+    (gogoproto.stdduration) = true,
+    (gogoproto.moretags) = "yaml:\"preserved_closed_sell_order_duration\""
+  ];
+
+  // Amount of time, if the SO expiry date enter the range before expiry of Dym-Name, the SO can not be placed.
```

Comment:
this is the recent description
```go
  // prohibit_sell_duration is the amount of time,
  // if the Sell-Order expiration date enters the range before expiry of Dym-Name, the Sell-Order can not be placed.
  // The logic also applies when the owner accepts a Buy-Order, the Dym-Name can not be sold.
  // This is to prevent the new owner of Dym-Name to sell it before the Dym-Name expires,
  // therefore help reduce the risk of new owner of Dym-Name not able to renew it on time.
  //
  // For example:
  //   Let say the configured Sell-Order duration is 7 days, the prohibit sell duration is 30 days and you owns a Dym-Name.
  //   We assume your Dym-Name expiration date is Mar 9th, 2024. then 30 days before, from Feb 9th, 2024,
  //   the Dym-Name is prohibited to be sold.
  //   We have some cases:
  //   - If the you places a Sell Order at Feb 2nd, 2024, the Sell-Order supposed to be expired at Feb 9th, 2024,
  //   which is within the prohibited range, so the Sell-Order can not be placed.
  //   - If the you places a Sell Order at Feb 1st, 2024, the Sell-Order supposed to be expired at Feb 8th, 2024,
  //   which is outside the prohibited range, so the Sell-Order can be placed.
  //   - If someone put an offer (Buy-Order) at Feb 1st, 2024, and you have some days to accept it,
  //   exactly from the moment order placed, until end of Feb 8th, 2024, which is outside the prohibited range.
  //   After that, from Feb 9th, it is prohibited to accept the Buy-Order.
```